lastfm/cli/lastfm.psl

```diff
@@ -0,0 +1,35 @@
+// Similarities like Pearson, Cosine, Adjusted Cosine Similarity between items.
```

Comment:
For all of the lists in this readme, use proper English.

E.G.
"Pearson, Cosine, Adjusted Cosine Similarity" -> "Pearson, Cosine, and Adjusted Cosine Similarity"
"Pearson, Cosine Similarity" -> "Pearson and Cosine Similarity"